src/js/components/CosmosErrorMessage.js

```diff
@@ -28,10 +28,14 @@ class CosmosErrorMessage extends React.Component {
   }
 
   getMessage() {
-    let {type, message} = this.props.error;
+    let {error} = this.props;
+    if (!error) {
+      return 'An error occurred (Description missing)';
```

Comment:
Could we change this message to something that is more sentence-like, perhaps `An unknown error occurred.` or something similar?

![](https://cl.ly/2w0g3I3l003s/Screen%20Shot%202016-08-24%20at%203.33.45%20PM.png)